DrinkACupOfJuice/html/jsn/1Q&A2n.js

```
@@ -122,11 +122,12 @@ let allQuestions = [];
                             // 创建一个新的div来包裹每个问题，并添加到问题列表中
                             const questionDiv = document.createElement('div');
 
-                            questionDiv.style.backgroundColor = 'white'; // 与页面主题一致的浅绿色背景
-                            questionDiv.style.padding = '10px'; // 内边距
-                            questionDiv.style.border = '1px solid #008000'; // 深绿色边框
-                            questionDiv.style.borderRadius = '5px'; // 边框圆角，与页面其他部分一致
-                            questionDiv.style.marginBottom = '10px'; // 底部外边距
+                            questionDiv.style.backgroundColor = '#333'; // 深灰色背景
+                            questionDiv.style.padding = '10px'; // 内边距保持不变
+                            questionDiv.style.border = '1px solid #666'; // 深灰色边框
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)